src/coreclr/nativeaot/System.Private.CoreLib/src/ILLink/ILLink.LinkAttributes.xml

```diff
@@ -14,10 +14,39 @@
   </assembly>
 
   <assembly fullname="*">
+
     <!-- Internal attributes shared as implementation, so they could be in any assembly -->
     <type fullname="System.Runtime.Versioning.NonVersionableAttribute">
       <attribute internal="RemoveAttributeInstances" />
     </type>
+
+    <!-- [EditorBrowsable] is only relevant for IDE scenarios -->
+    <type fullname="System.ComponentModel.EditorBrowsableAttribute">
+      <attribute internal="RemoveAttributeInstances" />
+    </type>
+
+    <!-- [CompilerGenerated] and [GeneratedCode] are often added by source generators, but don't really do anything -->
+    <type fullname="System.Runtime.CompilerServices.CompilerGeneratedAttribute">
+      <attribute internal="RemoveAttributeInstances" />
+    </type>
+    <type fullname="System.CodeDom.Compiler.GeneratedCodeAttribute">
+      <attribute internal="RemoveAttributeInstances" />
+    </type>
+
+    <!-- [ExcludeFromCodeAnalysis] is only relevant when doing local development or running tests -->
+    <type fullname="System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverageAttribute">
+      <attribute internal="RemoveAttributeInstances" />
+    </type>
+  </assembly>
+
+  <!-- Debugger support attributes, not used when debug support is disabled -->
+  <assembly fullname="*" feature="System.Diagnostics.Debugger.IsSupported" featurevalue="false">
```

Comment:
Out of curiosity, is it possible to also use multiple conditions for a given block? 🤔